doc(release): prepare release notes for Collect 23.04 (#2441)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md

## Centreon Collect

### 23.04.1

> Release date: `July 7, 2023`

Reviewer: ```suggestion
Release date: `July 5, 2023`
```

#### Centreon Broker

##### Bug fixes

- Changed the default value of `deletion_time` for acknowledgements to `null` to avoid displaying abnormal dates in the interface.

> - fix AcknowledgementType enum definition.

Reviewer 2: ```suggestion
```

- Fixed a bug that mainly occurred at backup time: a crash could occur when MariaDB was restarted.

> - Fixed an exception-catching issue that caused broker to fail inserting resources when the check_attempt was too high for the database column type.

Reviewer 2: ```suggestion
- Fixed an exception-catching issue that caused Broker to fail inserting resources when the check_attempt was too high for the database column type.
```

- Fixed the rebuilding of RRD databases that could result in incomplete graphs when duplicate values were present in `data_bin`. The duplicates are now skipped to avoid this.

##### Enhancements

> - On mariadb data in data_bin table are now inserted with a bulk prepared statement.

Reviewer 2: ```suggestion
- On MariaDB, data in data_bin table are now inserted with a bulk prepared statement.
```

> - On MariaDB, data in data_bin table are now inserted with a bulk prepared statement.

Reviewer: ```suggestion
- In MariaDB, data in the data_bin table are now inserted with a bulk prepared statement.
```

> - The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](../developer/developer-broker-stream-connector#the-broker_cache-object).

Reviewer: ```suggestion
- The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](../developer/developer-broker-stream-connector.md#the-broker_cache-object).
```

> - The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](../developer/developer-broker-stream-connector/#the-broker_cache-object).

Reviewer: ```suggestion
- The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](../developer/developer-broker-stream-connector#the-broker_cache-object).
```

> - The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](https://docs.centreon.com/docs/developer/developer-broker-stream-connector/#the-broker_cache-object).

Reviewer: ```suggestion
- The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](../developer/developer-broker-stream-connector/#the-broker_cache-object).
```

> - On mariadb data in data_bin table are inserted with a bulk prepared statement now.
> 
> ##### Enhancements
> 
> - The check_command had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](https://docs.centreon.com/docs/developer/developer-broker-stream-connector/#the-broker_cache-object).

Reviewer 2: ```suggestion

##### Enhancements

- On mariadb data in data_bin table are now inserted with a bulk prepared statement.
- The check_command field had been removed from Broker (host|service)_status events, making it temporarily unavailable for LUA stream connectors. It is now available in [Broker's cache](https://docs.centreon.com/docs/developer/developer-broker-stream-connector/#the-broker_cache-object).
```

- New performance counters have been added to the gRPC API in order to track the longest queries and statements.

#### Centreon Engine

##### Bug fixes

> - [Packaging] Filxed `/var/log/centreon-engine/archives` directory that was missing on Debian installations.

Reviewer: ```suggestion
- [Packaging] Added the `/var/log/centreon-engine/archives` directory that was missing on Debian installations.
```

> - Fixed RTDOWNTIMES external commands handling. The duration is not required anymore for fixed downtimes.

Reviewer: ```suggestion
- Fixed the handling of RTDOWNTIMES external commands. The duration is not required anymore for fixed downtimes.
```
